pages/your_problem/data_sharing.md

```diff
@@ -4,54 +4,73 @@ contributors: [Munazah Andrabi, Ulrike Witting, Elin Kronander]
+* Many repositories provide the option to put an embargo on a deposited dataset. This might be useful if you prefer to use the data in a publication before making it available for others to use.
+* FIX ME: Add point about collaboration agreement
 
-#### Subject Specific Repositories
-
-List of discipline-specific, community-recognized repositories can be found in the following links:
-
-* [ELIXIR Recommended Deposition Databases](https://elixir-europe.org/services/tag/elixir-deposition-databases).
-* [Nature Recommended Data Repositories](https://www.nature.com/sdata/policies/repositories).
-
-#### General Repositories
-List of Generalist repositories if no suitable community resource is available:
+## Which repository should I use to share my data?
+Once you have completed your experiments and have performed quality control of your data it is good scientific practice to share your data in a public repository. Sharing your data is often required by funders and publishers.
 
-* [4TU.Datacentrum](https://data.4tu.nl/info/en/).
-* [Dryad Digital Repository](http://datadryad.org).
-* [Figshare](https://figshare.com).
-* [Harvard Dataverse](https://dataverse.harvard.edu).
-* [Mendeley Data](https://data.mendeley.com).
-* [Open Science Framework](https://osf.io).
-* [Zenodo](https://zenodo.org).
-* [Code Ocean](https://codeocean.com).
-
-### Resources
+The most suitable repository will depend on the data type and your discipline.
 
+### Considerations
+  * What type of data are you planning to share?
+  * Does the repository need to provide solutions for restricted access for sensitive data?
+  * Do you have the rights to share the data via the repository?
+  * How sustainability is the repository, will the data remain public over time?
+  * How FAIR is the repository?
+  * Does the funding agency pose specific requirements regarding data sharing?
+  * FIX ME: Add point about licence/sharing policies of the repository
 
+### Solutions
+* Domain-Specific Repositories
+If a domain-specific repository, recognised by the community, exists this should be your first choice since domain-specific repositories often increases the FAIRness of the data.
+  * The [EMBL-EBI's data submission wizard](https://www.ebi.ac.uk/submission/) will help you choose a suitable repository based on your data type.
+  * Lists of domain-specific, community-recognised repositories can be found in the following links:
+    * [ELIXIR's Recommended Deposition Databases](https://elixir-europe.org/services/tag/elixir-deposition-databases).
+    * [scientific Data's Recommended Repositories](https://www.nature.com/sdata/policies/repositories).
+
+
+* General Repositories
+For other cases, your Institution or University may provide a centralised repository for storing the data or you may consider to use a general repository.
```

Comment:
I would consider to separate university repositories from general purpose repositories, since they are typically less FAIR than the general repositories. A local repo is good for internal sharing (and storing) but often not globally findable and accessible.